chain-extensions/xvm/src/lib.rs

```diff
@@ -76,28 +72,22 @@
                 // So we will charge a 32KB dummy value as a temporary replacement.
                 let charged_weight = env.charge_weight(weight_limit.set_proof_size(32 * 1024))?;
 
-                let caller = match env.ext().caller().clone() {
-                    Origin::Signed(address) => address,
-                    Origin::Root => {
-                        log::trace!(
-                            target: "xvm-extension::xvm_call",
-                            "root origin not supported"
-                        );
-                        return Ok(RetVal::Converging(
-                            XvmExecutionResult::from(CallError::BadOrigin).into(),
-                        ));
-                    }
-                };
+                let XvmCallArgs {
+                    vm_id,
+                    to,
+                    input,
+                    value,
+                } = env.read_as_unbounded(env.in_len())?;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code snippet introduces a new struct `XvmCallArgs` and assigns its fields from `env.read_as_unbounded(env.in_len())?`. However, the field `value` is missing in the old code. Please add the field `value` to the `XvmCallArgs` struct in the old code.
```diff
 let XvmCallArgs { vm_id, to, input, value } = env.read_as_unbounded(env.in_len())?;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->